src/wfs/src/main/java/org/geoserver/wfs/xml/filter/v1_1/PropertyNameTypeBinding.java

```diff
@@ -27,12 +26,9 @@
     /** the geoserver catalog */
     Catalog catalog;
 
-    /** parser namespace mappings */
-    NamespaceSupport namespaceSupport;
-
     public PropertyNameTypeBinding(
-            FilterFactory filterFactory, NamespaceSupport namespaceSupport, Catalog catalog) {
-        super(filterFactory);
+            FilterFactory2 filterFactory, NamespaceSupport namespaceSupport, Catalog catalog) {
+        super(filterFactory, namespaceSupport);
         this.namespaceSupport = namespaceSupport;
```

Comment:
Is this just me, or this assignement became useless?

```
        this.namespaceSupport = namespaceSupport;
```

Reply:
yes, that is true.